src/components/TimeCardPage/CommentModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react'
+//import { Button } from 'react-bootstrap';
+import {
+    Input, Button, useDisclosure, Modal,
+    ModalContent, ModalHeader, VStack, Card, CardBody, StackDivider, ModalFooter
+}
+    from '@chakra-ui/react'
+import { useFormik } from "formik";
+
+export default function CommentModal(props) {
+
+    // enum for the type of comment the manager is leaving
+    const CommentType = {
+        Report: "Rejected",
+        Comment: "Comment"
+    }
+
+    const formik = useFormik({
+        initialValues: {
+            type: "",
+            remarks: "",
+        },
+        onSubmit: (values) => {
+            // TODO: utilize a POST request to send comment to the backend (data is stored in 'values')
+            alert(JSON.stringify(values, null, 2));
+        }
+    });
+
+    const { isOpen, onOpen, onClose } = useDisclosure()
+
+    return (
+        <>
+            <Button onClick={onOpen}>Comments/Reports</Button>
+            <Modal
+                isOpen={isOpen}
+                onClose={onClose}
+                isCentered
+                colorScheme={"red"}
+            >
+                <ModalContent>
+                    <VStack spacing={4} divider={<StackDivider />}>
+                        <ModalHeader>Leave a comment/report</ModalHeader>
+                        <form onSubmit={formik.handleSubmit}>
+                            <label htmlFor="remarks">Remarks</label>
```

Comment:
Look into using Chakra's `FormControl` and `FormLabel` for consistency with styling https://chakra-ui.com/docs/components/form-control